packages/mask/src/plugins/MaskBox/SNSAdaptor/components/PreviewCard.tsx

```diff
@@ -12,32 +12,13 @@ import { DrawResultDialog } from './DrawResultDialog'
 import { useTransactionCallback, TargetChainIdContext } from '@masknet/plugin-infra/web3-evm'
 import { ChainBoundary } from '../../../../web3/UI/ChainBoundary'
 import { formatBalance, NetworkPluginID } from '@masknet/web3-shared-base'
-import type { AbstractTabProps } from '../../../../components/shared/AbstractTab'
-import AbstractTab from '../../../../components/shared/AbstractTab'
+import { TabContext } from '@mui/lab'
+import { formatAddress } from '@masknet/web3-shared-flow'
```

Comment:
At least use `formatAddress` for EVM.
```suggestion
import { formatAddress } from '@masknet/web3-shared-evm'
```